env_canada/ec_aqhi.py

```diff
@@ -152,7 +152,7 @@ async def get_aqhi_data(self, url):
                 return None
 
             result = await response.read()
-            aqhi_xml = result.decode("ISO-8859-1")
+            aqhi_xml = result
```

Comment:
Why did you change this? I seem to recall it being required to handle characters with French accents.

Reply:
The lxml etree is looking at the header and uses the coding specified there to decode. I will pull all the aqhi data in both languages and see if it fails at all. Some spot checks I did seem fine.
http://dd.weather.gc.ca/air_quality/aqhi/pyr/observation/realtime/xml/AQ_OBS_KAHFT_CURRENT.xml
```xml
<?xml version='1.0' encoding='utf-8'?>

Reply:
I wrote a script and turned on debug. I pulled down all the current and forecasts fro all the sites. The only errors were some 404s for sites that didn't have current observations. 

```python
import asyncio
from datetime import datetime

import pytest

from env_canada import ECAirQuality, ec_aqhi
from env_canada.ec_aqhi import get_aqhi_regions

import logging
logger = logging.getLogger(__name__)
logging.basicConfig(filename='aqhi.log', encoding='utf-8', level=logging.DEBUG)


def main(*args,**kwargs):
    all_regions = get_aqhi_regions("FR")
    regions = asyncio.run(all_regions)
    for reg in regions:
        aqhi = ECAirQuality(coordinates=(reg["latitude"], reg["longitude"]), language="FR")
        asyncio.run(aqhi.update())
        print(aqhi.zone_id)
        print(aqhi.region_id)
        print(aqhi.metadata)
        print(aqhi.current)
        print(aqhi.forecasts)

if __name__ == "__main__":
    main()